### ISSUE TITLE
run/vector1.scala fails to find implicit

### ISSUE THREAD

**Issue by DarkDimius** [lampepfl/dotty#681]
```
[info] compiling /Users/dark/workspace/dotty/tests/partest-generated/run/vector1.scala
[info] options: -pagewidth 160 -Yno-deep-subtypes -Yno-double-bindings -Ycheck:tailrec,resolveSuper,mixin,restoreScopes -d ./tests/partest-generated/run/vector1-run.obj
[info] /Users/dark/workspace/dotty/tests/partest-generated/run/vector1.scala:128: error: no implicit argument of type scala.collection.generic.CanBuildFrom[
[info]   (CC? >: ([HK$0] => scala.collection.mutable.WrappedArray[HK$0])[HK$0 = Int] <: ([HK$0] =>  <: scala.collection.TraversableOnce[HK$0]))[(T? = Int)]#Apply
[info] , (T? = Int),
[info]   (CC? >: ([HK$0] => scala.collection.mutable.WrappedArray[HK$0])[HK$0 = Int] <: ([HK$0] =>  <: scala.collection.TraversableOnce[HK$0]))[(T? = Int)]#Apply
[info] ] found for parameter bf of method shuffle in class Random
[info]     val pos = scala.util.Random.shuffle(scala.collection.mutable.WrappedArray.make[Int](Array.tabulate[Int](N)(i => i)))
[info]                                                                                                                         ^
```

**Comment by smarter** [comment]
Probably duplicate of https://github.com/lampepfl/dotty/issues/553

**Comment by smarter** [comment]
Fixed by #799, leaving this open until we move the test out of `pending`.

**Comment by odersky** [comment]
@smarter I think it's preferable to move the test out of pending immediately if that can close an issue. If we do not do this, issues stay open and it takes unnecessary effort to come back to the context.
